lib/configuration/variables/sources/opt.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const ensureString = require('type/string/ensure');
+const ServerlessError = require('../../../serverless-error');
+
+module.exports = {
+  resolve: ({ address, options }) => {
+    if (!address) {
+      throw new ServerlessError(
+        'Missing address argument in variable "opt" source',
```

Comment:
According to our current docs, it's possible to use `${opt:}` to get the whole object - are we deprecating that?

Reply:
Great point, addressed locally (to be pushed)